.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: c
 sudo: required
 services: docker
-script: bash ./.travis-$BUILD_METHOD.sh
+script:
+    - bash ./.travis-$BUILD_METHOD.sh
+    - nosetests scripts/ scripts/examples scripts/examples/python
```

Comment:
I think this means that we will run the Python tests twice, both for the `BUILD_METHOD=xenserver-build-env` and `BUILD_METHOD=opam-coverage` jobs. Maybe we could create another job for this, and a script `.travis-python-tests`?